src/components/views/styl/MonitoringChart.styl

```diff
@@ -0,0 +1,12 @@
+@import 'includes/variables';
+
+.monitoring-charts {
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    border-bottom: 1px solid $gray;
+
+    > *:first-child {
```

Comment:
Nitpick, but the `*` is superflous; `> :first-child {` does the same thing.

Reply:
Neat. Really didn't realize that.

Reply:
I had to look it up myself. Here's what [the spec](https://www.w3.org/TR/selectors-3/#universal-selector) says:

> If a universal selector represented by * ... is immediately followed by a pseudo-element, then the * may be omitted and the universal selector's presence implied.

But they also say:

> Note: it is recommended that the * not be omitted, because it decreases the potential confusion between, for example, div :first-child and div:first-child. Here, div *:first-child is more readable. 

So maybe your way is best.